### PR TITLE
[PR] Add batch to `true`

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).

### CODE REVIEW

File: build-system/pr-validation.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Pull request validation for Windows against the `dev` and `master` branches
 # See https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema for reference
 trigger:
+  batch: true
```
Comment (eaba) [inline build-system/pr-validation.yaml:4]: image
https://github.com/microsoft/azure-pipelines-yaml/blob/master/design/pipeline-triggers.md

**Review by eaba — COMMENTED** [review]
Added batch to true

File: build-system/pr-validation.yaml (hunk 1 above)
Comment (eaba) [reply build-system/pr-validation.yaml:4]: more examples
https://github.com/Azure/data-api-builder/blob/e810143c6bbe58d056181dda32b3703ea2ebb8ad/.pipelines/azure-pipelines.yml#L4-L11

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**